bigquery/cloud-client/query.py

```diff
@@ -63,6 +63,30 @@ def query_standard_sql(query):
         print(row)
 
 
+def query_destination_table(query, dest_dataset_id, dest_table_id):
+    client = bigquery.Client()
+    query_job = client.run_async_query(str(uuid.uuid4()), query)
+
+    # Allow for query results larger than the maximum response size.
+    query_job.allow_large_results = True
+
+    # When large results are allowed, a destination table must be set.
+    dest_dataset = client.dataset(dest_dataset_id)
+    dest_table = dest_dataset.table(dest_table_id)
+    query_job.destination = dest_table
+
+    # Allow the results table to be overwritten.
+    query_job.write_disposition = 'WRITE_TRUNCATE'
+
+    query_job.begin()
```

Comment:
doesn't .result() call .begin()?

Reply:
Oh, I didn't realize that. I can update the sample if so.